### PR TITLE
update doc and codes for GLDAS

### PR DESCRIPTION
**Pull request by tslin2**
Revise for GLDAS climate forcing, particularly for TabError: inconsistent use of tabs and spaces in indentation using python.

### CODE REVIEW

File: hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas.py [hunk 1]
```diff
@@ -38,30 +38,30 @@
             modays = leap_days
         if (yrs[yy] == "00" or "04" or "08" or "12" or "16"):
             modays = leap_days
-        for julday in range(day_start+1,day_end+2,1):
+        for julday in range(day_start,day_end+1,1):
             for mo in range(0,12,1):
                 if (julday>modays[mo] and julday<=modays[mo+1]):
-		    day = (julday - modays[mo])
+                    day = (julday - modays[mo])
```
Comment (cenlinhe) [inline hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas.py:44]: This new "green" line seems exactly the same as the original one deleted "red" here. What has changed? Similar for the two places below?

**Review by CharlesZheZhang — APPROVED** [review]
Hello Tzu-Shun, thank you for correcting the Tab errors in these python codes!

File: hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas_init.py [hunk 2]
```diff
@@ -20,12 +21,12 @@
 
 for var in range(len(vars_name)):
     if vars_name[var] == "SoilMoi" or vars_name[var] == "SoilTMP":
-        os.system("ncks -v "+vars_name[var]+"0_10cm_inst " +filename+" "+ results_dir+"/GLDAS_"+vars_name[var]+"_000-010_"+date+"00")
-        os.system("ncks -v "+vars_name[var]+"10_40cm_inst " +filename+" "+ results_dir+"/GLDAS_"+vars_name[var]+"_010-040_"+date+"00")
-        os.system("ncks -v "+vars_name[var]+"40_100cm_inst " +filename+" "+ results_dir+"/GLDAS_"+vars_name[var]+"_040-100_"+date+"00")
-        os.system("ncks -v "+vars_name[var]+"100_200cm_inst " +filename+" "+ results_dir+"/GLDAS_"+vars_name[var]+"_100-200_"+date+"00")
+        os.system("ncks -v "+vars_name[var]+"0_10cm_inst " +filename+" "+ results_dir+"/INIT/GLDAS_"+vars_name[var]+"_000-010_"+date+"00")
+        os.system("ncks -v "+vars_name[var]+"10_40cm_inst " +filename+" "+ results_dir+"/INIT//GLDAS_"+vars_name[var]+"_010-040_"+date+"00")
```
Comment (cenlinhe) [inline hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas_init.py]: There is an repeated "/" after "INIT".

**Comment by cenlinhe** [comment]
Have you tested this corrected GLDAS code? Does it successfully run and solve the problem?

File: hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas.py (hunk 1 above)
Comment (CharlesZheZhang) [reply hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas.py:44]: Hi Cenlin, the changes are related to consistent use of tabs or spaces in python, i.e. "TabError: inconsistent use of tabs and spaces in indentation using python".

Comment (cenlinhe) [reply hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas.py:44]: Oh, I see. thanks! Is this a change to be compatible with the latest python version (e.g., after v3)? Would this also work with the older python version (e.g., v2.7)?

Comment (CharlesZheZhang) [reply hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas.py:44]: The usage of tabs and spaces for indenting in one file is not allowed in python3, but allowed in python2. That's why some users with python3 had this issue before. Now should be OK.

File: hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas_init.py (hunk 2 above)
Comment (tslin2) [reply hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas_init.py]: I just revised it, thanks

**Comment by tslin2** [comment]
> Have you tested this corrected GLDAS code? Does it successfully run and solve the problem?

A case is tested between 2012/02/27 and 2012/03/01. It runs successfully.

**Review by cenlinhe — APPROVED** [review]
The changes look good.

**Pull request merged**